app/src/main/java/com/gdsctsec/smartt/ui/edit/EditScreenActivity.kt

```diff
@@ -0,0 +1,176 @@
+package com.gdsctsec.smartt.ui.edit
+
+import android.app.TimePickerDialog
+import android.os.Bundle
+import android.text.Editable
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import com.gdsctsec.smartt.databinding.ActivityEditscreenBinding
+import java.util.*
+import android.graphics.Color
+
+import android.text.TextWatcher
+import android.widget.*
+import com.gdsctsec.smartt.R
+import com.google.android.material.textfield.TextInputEditText
+import com.google.android.material.textfield.TextInputLayout
+import android.content.Intent
+
+import android.app.Activity
+
+import android.R.attr.name
+
+
+
+
+
+class EditScreenActivity : AppCompatActivity() {
+
+    private lateinit var binding : ActivityEditscreenBinding
+    // Initializing
+
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityEditscreenBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
+        val lectureEditText : EditText = binding.textinputAddlecture
+        val starttimeTextView : TextView = binding.textviewStarttime
+        val EndtimeTextView : TextView = binding.textviewEndtime
+        val dayTextInputEditText : AutoCompleteTextView = binding.textfieldDay
+        val SaveTextview : TextView = binding.textViewSave
+        val cancelTextView : TextView  = binding.textViewCancel
+
+
+
+
+        // dropdown
+        var days = resources.getStringArray(com.gdsctsec.smartt.R.array.days)
+        val ArrayAdapter = ArrayAdapter(
+            applicationContext,
```

Comment:
you don't need an applicationContext here, try replacing it with `requireActivity()`